src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/QuestionnaireAnswer.kt

```diff
@@ -0,0 +1,78 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.Column
+import jakarta.persistence.Convert
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Generated
+import org.hibernate.type.YesNoConverter
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "QUESTIONNAIRE_ANSWERS")
+data class QuestionnaireAnswer(
+  @Id
+  @Column(name = "QUESTIONNAIRE_ANS_ID")
+  @SequenceGenerator(name = "QUESTIONNAIRE_ANS_ID", sequenceName = "QUESTIONNAIRE_ANS_ID", allocationSize = 1)
+  @GeneratedValue(generator = "QUESTIONNAIRE_ANS_ID")
+  var id: Long = 0,
+
+  @Column(name = "DESCRIPTION")
+  val answer: String,
+
+  @ManyToOne(fetch = FetchType.LAZY, optional = true)
+  @JoinColumn(name = "NEXT_QUESTIONNAIRE_QUE_ID")
+  var nextQuestion: QuestionnaireQuestion? = null,
+
+  @Column(name = "ACTIVE_FLAG", nullable = false)
+  @Convert(converter = YesNoConverter::class)
+  val active: Boolean = true,
+
+  @Column(name = "ANS_SEQ")
+  val answerSequence: Int,
+
+  @Column(name = "LIST_SEQ")
+  val listSequence: Int,
+
+  @Column(name = "COMMENT_REQUIRED_FLAG", nullable = false)
+  @Convert(converter = YesNoConverter::class)
+  val commentRequired: Boolean = false,
+
+  @Column(name = "DATE_REQUIRED_FLAG", nullable = false)
+  @Convert(converter = YesNoConverter::class)
+  val dateRequired: Boolean = false,
+
+  @Column
+  var auditModuleName: String? = null,
+
+  @Column(name = "CREATE_DATETIME", nullable = false, insertable = false, updatable = false)
```

Comment:
same here about generated - maybe drop updated? I doubt it will ever be used so allow DB defaults for that

Reply:
dropped all updated - can easily add if needed. Yep agree re generated.